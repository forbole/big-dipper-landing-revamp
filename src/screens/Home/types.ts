/**
 * `HomeProps` is an object with a property called `networkList` that is an array of `Network` objects.
 * @property networkList - Network[];
 */
export interface HomeProps {
  networkList: Network[];
}
