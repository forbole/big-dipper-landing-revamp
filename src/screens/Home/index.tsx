import KeyboardArrowLeftIcon from '@mui/icons-material/KeyboardArrowLeft';
import KeyboardArrowRightIcon from '@mui/icons-material/KeyboardArrowRight';
import Box from '@mui/material/Box';
import Fab from '@mui/material/Fab';
import classnames from 'classnames';
import useTranslation from 'next-translate/useTranslation';
import Head from 'next/head';
import { FC, LegacyRef, useCallback, useRef, useState } from 'react';
import Slider from 'react-slick';
import { Layout } from '@/src/components';
import SectionBox from '@/src/components/SectionBox';
import SectionLimit from '@/src/components/SectionLimit';
import NetworksPanel from './components/NetworksPanel';
import Slide1 from './components/Slide1';
import Slide2 from './components/Slide2';
import Slide3 from './components/Slide3';
import Slide4 from './components/Slide4';
import type { HomeProps } from './types';
import useStyles from './useStyles';
/* A React component that renders the Homepage. */
const Home: FC<HomeProps> = ({ networkList }) => {
  const sliderRef: LegacyRef<Slider> = useRef(null);
  const handleSlideLeft = useCallback(() => {
    sliderRef.current?.slickPrev();
  }, []);
  const handleSlideRight = useCallback(() => {
    sliderRef.current?.slickNext();
  }, []);
  const [sliding, setSliding] = useState(false);
  const [slide, setSlide] = useState(0);
  const beforeChange = useCallback(() => setSliding(true), []);
  const afterChange = useCallback((current: number) => {
    setSlide(current);
    setSliding(false);
  }, []);
  const { t } = useTranslation('common');
  const styles = useStyles();

  return (
    <Layout css={styles.layout}>
      <Head>
        <title>{t('bigDipper')}</title>
      </Head>
      <div className="home_slider-container" css={[styles.slider]}>
        <Fab className="home__slider-left" onClick={handleSlideLeft}>
          <KeyboardArrowLeftIcon />
        </Fab>
        <Fab className="home__slider-right" onClick={handleSlideRight}>
          <KeyboardArrowRightIcon />
        </Fab>
        <Box
          className={classnames({
            [`home__slide${slide}`]: true,
            home__sliding: sliding,
          }, 'home__slider')}
        >
          <Slider
            dots
            slidesToScroll={1}
            slidesToShow={1}
            arrows={false}
            beforeChange={beforeChange}
            afterChange={afterChange}
            ref={sliderRef}
            useTransform={false}
          >
            <Slide1 />
            <Slide2 />
            <Slide3 />
            <Slide4 />
          </Slider>
        </Box>
      </div>
      <SectionBox className="home__section-box" css={styles.root}>
        <SectionLimit>
          <NetworksPanel networkList={networkList} />
        </SectionLimit>
      </SectionBox>
    </Layout>
  );
};

export default Home;
