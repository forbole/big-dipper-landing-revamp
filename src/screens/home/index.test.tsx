import React from 'react';
import { render } from '@testing-library/react';
import { MockTheme } from '@mocks';
import Home from '.';

// ==================================
// mocks
// ==================================
const mockI18n = {
  t: (key: string) => key,
  lang: 'en',
};
jest.mock('next-translate/useTranslation', () => () => mockI18n);

// ==================================
// unit tests
// ==================================
describe('Dummy Test', () => {
  it('matches snapshot', async () => {
    const { asFragment, getByAltText } = render(
      <MockTheme><Home /></MockTheme>,
    );
    expect(getByAltText(/^Big Dipper$/)).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });

  afterEach(() => {
    jest.clearAllMocks();
  });
});
