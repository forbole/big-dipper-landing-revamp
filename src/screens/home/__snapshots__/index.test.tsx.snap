// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dummy Test matches snapshot 1`] = `
<DocumentFragment>
  @keyframes animation-0 {
  50% {
    -webkit-transform: scale(0.98);
    -moz-transform: scale(0.98);
    -ms-transform: scale(0.98);
    transform: scale(0.98);
  }
}

.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  min-height: 100vh;
  background-image: radial-gradient(300vw 100v at 50% 50vh, rgba(212,112,151,0), #30177d),url(/images/background/back.png);
  background-repeat: no-repeat;
  -webkit-background-position: 50% 0;
  background-position: 50% 0;
}

@media (min-width:768px) {
  .emotion-0 {
    background-image: radial-gradient(512px 512px at 50% 300px, rgba(212,112,151,0), #30177d),url(/images/background/back.png);
  }
}

.emotion-1 {
  margin-left: auto;
  margin-right: auto;
  max-width: 1024px;
  position: relative;
  width: 100%;
}

.emotion-1>button {
  height: 60px;
  position: absolute;
  padding: 40px;
  z-index: 1;
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex-flow: row nowrap;
  -webkit-flex-flow: row nowrap;
  -ms-flex-flow: row nowrap;
  flex-flow: row nowrap;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 79px;
  padding: 16px;
}

@media (min-width:768px) {
  .emotion-2 {
    -webkit-box-pack: justify;
    -webkit-justify-content: space-between;
    justify-content: space-between;
  }
}

.emotion-2 .menubar .MuiButtonBase-root {
  border-radius: 24px;
}

.emotion-2 .menubar .submenu-container .MuiButtonBase-root {
  border-radius: 0;
}

.emotion-2 .menubar .submenu-container .MuiButtonBase-root:first-of-type {
  border-top-left-radius: 16px;
  border-top-right-radius: 16px;
}

.emotion-2 .menubar .submenu-container .MuiButtonBase-root:last-of-type {
  border-bottom-left-radius: 16px;
  border-bottom-right-radius: 16px;
}

.emotion-3 {
  height: 36px;
  display: block;
}

@media (min-width:768px) {
  .emotion-3 {
    margin: 8px;
  }
}

.emotion-4 {
  background: unset;
  border: unset;
  height: 60px;
  width: 60px;
  position: absolute;
  right: 0px;
  cursor: pointer;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  z-index: 2;
}

.emotion-4 .bar1,
.emotion-4 .bar2,
.emotion-4 .bar3 {
  width: 24px;
  border: 1px solid #FFFFFF;
  margin: 6px 0;
  -webkit-transition: 0.4s;
  transition: 0.4s;
  border-radius: 25px;
}

.emotion-4 .hamburger-X .bar2 {
  opacity: 0;
}

.emotion-4 .hamburger-X .bar1 {
  -webkit-transform: rotate(-45deg) translate(-4.5px, 4.5px);
  -moz-transform: rotate(-45deg) translate(-4.5px, 4.5px);
  -ms-transform: rotate(-45deg) translate(-4.5px, 4.5px);
  transform: rotate(-45deg) translate(-4.5px, 4.5px);
}

.emotion-4 .hamburger-X .bar3 {
  -webkit-transform: rotate(45deg) translate(-6.5px, -6.5px);
  -moz-transform: rotate(45deg) translate(-6.5px, -6.5px);
  -ms-transform: rotate(45deg) translate(-6.5px, -6.5px);
  transform: rotate(45deg) translate(-6.5px, -6.5px);
}

@media (min-width:768px) {
  .emotion-4 {
    display: none;
  }
}

.emotion-5 {
  display: none;
  width: auto;
  -webkit-box-pack: right;
  -ms-flex-pack: right;
  -webkit-justify-content: right;
  justify-content: right;
}

@media (min-width:768px) {
  .emotion-5 {
    display: -webkit-inline-box;
    display: -webkit-inline-flex;
    display: -ms-inline-flexbox;
    display: inline-flex;
  }
}

.emotion-5>.MuiList-root>.MuiListItemButton-root:first-of-type {
  display: none!important;
}

.emotion-5 .MuiList-root {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-5 .MuiButtonBase-root {
  padding: 8px 24px;
}

.emotion-5 .submenu,
.emotion-5 .active {
  background: transparent none!important;
}

.emotion-5 .active.submenu {
  background: linear-gradient(103.18deg, #F6504B 20.77%, #DC001A 118.69%)!important;
}

.emotion-5 .submenu-box {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  position: relative;
}

.emotion-5 .submenu-container {
  position: absolute;
  top: 100%;
  width: 100%;
  margin-top: 16px;
  background: #3c52af none;
  border-radius: 16px;
  z-index: 10;
}

.emotion-5 .submenu-container .MuiButtonBase-root {
  padding: 16px 24px;
}

.emotion-6 {
  list-style: none;
  margin: 0;
  padding: 0;
  position: relative;
  padding-top: 8px;
  padding-bottom: 8px;
}

.emotion-7 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  -webkit-box-pack: start;
  -ms-flex-pack: start;
  -webkit-justify-content: flex-start;
  justify-content: flex-start;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  position: relative;
  -webkit-text-decoration: none;
  text-decoration: none;
  min-width: 0;
  box-sizing: border-box;
  text-align: left;
  padding-top: 8px;
  padding-bottom: 8px;
  -webkit-transition: background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  padding-left: 16px;
  padding-right: 16px;
  text-align: left;
  padding: 8px 32px;
}

.emotion-7::-moz-focus-inner {
  border-style: none;
}

.emotion-7.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-7 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-7:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: rgba(255, 255, 255, 0.08);
}

@media (hover: none) {
  .emotion-7:hover {
    background-color: transparent;
  }
}

.emotion-7.Mui-selected {
  background-color: rgba(144, 202, 249, 0.16);
}

.emotion-7.Mui-selected.Mui-focusVisible {
  background-color: rgba(144, 202, 249, 0.28);
}

.emotion-7.Mui-selected:hover {
  background-color: rgba(144, 202, 249, 0.24);
}

@media (hover: none) {
  .emotion-7.Mui-selected:hover {
    background-color: rgba(144, 202, 249, 0.16);
  }
}

.emotion-7.Mui-focusVisible {
  background-color: rgba(255, 255, 255, 0.12);
}

.emotion-7.Mui-disabled {
  opacity: 0.38;
}

.emotion-7.MuiListItem-gutters {
  padding: 16px 20px;
}

.emotion-7 .MuiListItemText-root {
  color: #fff;
  font-weight: 900;
}

.emotion-7.active {
  background: linear-gradient(103.18deg, #F6504B 20.77%, #DC001A 118.69%);
}

.emotion-7.submenu {
  background: #3c52af none;
}

.emotion-8 {
  -webkit-flex: 1 1 auto;
  -ms-flex: 1 1 auto;
  flex: 1 1 auto;
  min-width: 0;
  margin-top: 4px;
  margin-bottom: 4px;
}

.emotion-9 {
  margin: 0;
  font-size: 1rem;
  white-space: pre-wrap;
  font-family: SFPro,Arial,sans-serif;
  font-weight: 400;
  line-height: 1.5;
  display: block;
}

.emotion-10 {
  overflow: hidden;
  pointer-events: none;
  position: absolute;
  z-index: 0;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  border-radius: inherit;
}

.emotion-19 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  font-size: 1.5rem;
}

.emotion-30 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.emotion-31 {
  overflow: auto;
  padding: 48px 24px;
  padding: 0;
  overflow: visible;
}

@media (min-width:1280px) {
  .emotion-31.main {
    padding: 64px;
  }
}

.emotion-32 {
  margin-left: auto;
  margin-right: auto;
  height: 500px;
}

.emotion-38 {
  margin-left: auto;
  margin-right: auto;
  max-width: 1024px;
  padding: 40px 32px;
}

.emotion-38 .icons__wrapper path {
  fill: #fff;
}

.emotion-38 .icons__wrapper {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  padding: 0px 16px;
  margin-bottom: 40px;
}

.emotion-38 .terms__wrapper {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-38 .MuiTypography-caption {
  font-weight: 500;
  color: #fff;
}

.emotion-38 .terms__links {
  margin: 8px 0px;
}

.emotion-38 .terms--dash {
  margin: 0px 8px;
}

.emotion-38 .terms--dash.desktop {
  display: none;
}

@media (min-width:768px) {
  .emotion-38 .icons__wrapper {
    -webkit-box-pack: center;
    -ms-flex-pack: center;
    -webkit-justify-content: center;
    justify-content: center;
  }

  .emotion-38 .social {
    margin: 0px 12px;
  }
}

@media (min-width:1280px) {
  .emotion-38 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    -webkit-box-pack: justify;
    -webkit-justify-content: space-between;
    justify-content: space-between;
  }

  .emotion-38 .icons__wrapper {
    padding-left: 0;
    margin-bottom: 0;
  }

  .emotion-38 .social:first-of-type {
    margin-left: 0;
  }

  .emotion-38 .terms__wrapper {
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
  }

  .emotion-38 .terms--dash.desktop {
    display: inline;
  }
}

.emotion-39 {
  margin: 0;
  font-size: 0.75rem;
  font-family: SFPro,Arial,sans-serif;
  font-weight: 400;
  line-height: 1.66;
}

<main
    class="emotion-0"
  >
    <div>
      <div
        class="emotion-1"
      >
        <div
          class="wrapper"
        >
          <span
            class="logo"
            height="36"
          />
          <button
            class="emotion-2"
            type="button"
          >
            <div
              class="hamburger"
            >
              <div
                class="bar1"
              />
              <div
                class="bar2"
              />
              <div
                class="bar3"
              />
            </div>
          </button>
        </div>
        <div
          class="menubar MuiBox-root emotion-5"
        >
          <ul
            class="MuiList-root MuiList-padding emotion-6"
            role="menu"
            tabindex="-1"
          >
            <a
              class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters  emotion-7"
              href="/"
              tabindex="0"
            >
              <div
                class="MuiListItemText-root emotion-8"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary emotion-9"
                >
                  home
                </span>
              </div>
              <span
                class="MuiTouchRipple-root emotion-10"
              />
            </a>
            <a
              class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters  emotion-7"
              href="/all-networks"
              tabindex="0"
            >
              <div
                class="MuiListItemText-root emotion-8"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary emotion-9"
                >
                  allNetworks
                </span>
              </div>
              <span
                class="MuiTouchRipple-root emotion-10"
              />
            </a>
            <div
              class="submenu-box MuiBox-root emotion-15"
            >
              <div
                class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters submenu emotion-7"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiListItemText-root emotion-8"
                >
                  <span
                    class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary emotion-9"
                  >
                    helpsAndSupport
                  </span>
                </div>
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium emotion-19"
                  data-testid="KeyboardArrowDownIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7.41 8.59 12 13.17l4.59-4.58L18 10l-6 6-6-6 1.41-1.41z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root emotion-10"
                />
              </div>
              <div
                class="submenu-container MuiBox-root emotion-15"
              />
            </div>
            <a
              class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters  emotion-7"
              href="/about"
              tabindex="0"
            >
              <div
                class="MuiListItemText-root emotion-8"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary emotion-9"
                >
                  about
                </span>
              </div>
              <span
                class="MuiTouchRipple-root emotion-10"
              />
            </a>
            <a
              class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters  emotion-7"
              href="https://docs.bigdipper.live/"
              tabindex="0"
            >
              <div
                class="MuiListItemText-root emotion-8"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary emotion-9"
                >
                  docs
                </span>
              </div>
              <span
                class="MuiTouchRipple-root emotion-10"
              />
            </a>
          </ul>
        </div>
      </div>
    </div>
    <div
      class="emotion-30"
    >
      <section
        class=" emotion-31"
      >
        <div
          class="emotion-32"
        >
          <div
            class="logo"
          >
            <div
              class="slider MuiBox-root emotion-15"
            >
              <div
                class="slick-slider slick-initialized"
              >
                <div
                  class="slick-list"
                >
                  <div
                    class="slick-track"
                    style="opacity: 1; transform: translate3d(0px, 0px, 0px);"
                  >
                    <div
                      aria-hidden="false"
                      class="slick-slide slick-active slick-current"
                      data-index="0"
                      style="outline: none; width: 0px;"
                      tabindex="-1"
                    >
                      <div>
                        <div
                          class="MuiBox-root emotion-34"
                        >
                          <h2
                            class="MuiTypography-root MuiTypography-h2 emotion-35"
                          >
                            Read-time Interchain Explorer
                          </h2>
                          <h4
                            class="MuiTypography-root MuiTypography-h4 emotion-36"
                          >
                            View real-time status of different networks
                          </h4>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiBox-root emotion-37"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27482%27%20height=%27452%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  alt="Big Dipper"
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
          </div>
        </div>
      </section>
    </div>
    <footer>
      <div
        class="emotion-38"
      >
        <div
          class="icons__wrapper"
        >
          <a
            class="social twitter"
            href="https://twitter.com/bigdipperlive"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              fill="none"
              height="12"
              viewBox="0 0 14 12"
              width="14"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M12.33 3.23897C12.3353 3.35997 12.3353 3.47547 12.3353 3.59647C12.3406 7.26499 9.64538 11.5 4.71643 11.5C3.26269 11.5 1.83547 11.0655 0.609863 10.2515C0.822089 10.279 1.03431 10.29 1.24654 10.29C2.45092 10.29 3.62347 9.87199 4.57318 9.09649C3.42716 9.07449 2.41909 8.29899 2.06892 7.16599C2.47215 7.24849 2.88599 7.23199 3.2786 7.11649C2.03178 6.86349 1.13512 5.72498 1.12982 4.39948C1.12982 4.38848 1.12982 4.37748 1.12982 4.36648C1.50121 4.58098 1.92036 4.70198 2.34481 4.71298C1.17226 3.89898 0.806172 2.27647 1.51713 1.00597C2.88068 2.74397 4.88621 3.79448 7.04031 3.90998C6.82277 2.94747 7.11989 1.93547 7.81493 1.25347C8.89198 0.202964 10.5898 0.257964 11.6085 1.37447C12.208 1.25347 12.7863 1.02247 13.3116 0.697965C13.11 1.34147 12.6908 1.88597 12.1337 2.23247C12.6643 2.16647 13.1842 2.01797 13.6724 1.79797C13.3116 2.35897 12.8553 2.84297 12.33 3.23897Z"
                fill="#FCCDCC"
              />
            </svg>
          </a>
          <a
            class="social telegram"
            href="https://t.me/forbole"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              fill="none"
              height="12"
              viewBox="0 0 16 12"
              width="16"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clip-rule="evenodd"
                d="M1.89631 5.16592C5.78325 3.47245 8.37516 2.356 9.67202 1.81659C13.3748 0.276465 14.1442 0.00892841 14.6457 9.41924e-05C14.756 -0.00184879 15.0027 0.0254864 15.1624 0.155112C15.2973 0.264566 15.3344 0.412421 15.3522 0.516195C15.3699 0.619969 15.392 0.856369 15.3745 1.04108C15.1738 3.1494 14.3056 8.26572 13.8639 10.6271C13.677 11.6262 13.3089 11.9613 12.9527 11.9941C12.1784 12.0653 11.5904 11.4823 10.8405 10.9908C9.66695 10.2215 9.00399 9.74265 7.86489 8.992C6.54847 8.1245 7.40185 7.64771 8.15208 6.86849C8.34841 6.66457 11.76 3.56151 11.826 3.28001C11.8343 3.2448 11.8419 3.11357 11.764 3.04428C11.686 2.97498 11.5709 2.99868 11.4879 3.01753C11.3702 3.04424 9.4955 4.28334 5.86381 6.73483C5.33168 7.10023 4.8497 7.27826 4.41786 7.26893C3.94179 7.25865 3.02602 6.99976 2.34524 6.77846C1.51024 6.50704 0.846598 6.36353 0.904387 5.90257C0.934487 5.66247 1.26513 5.41692 1.89631 5.16592Z"
                fill="#FCCDCC"
                fill-rule="evenodd"
                id="Vector"
              />
            </svg>
          </a>
          <a
            class="social github"
            href="https://github.com/forbole"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              fill="none"
              height="16"
              viewBox="0 0 18 16"
              width="18"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clip-rule="evenodd"
                d="M9.1384 0C7.18005 0.00101608 5.28592 0.694626 3.79469 1.9568C2.30346 3.21898 1.31238 4.96742 0.998662 6.88949C0.684944 8.81155 1.06904 10.7819 2.08229 12.4482C3.09553 14.1145 4.67184 15.3682 6.52935 15.9849C6.93915 16.061 7.09351 15.807 7.09351 15.5911C7.09351 15.3751 7.08531 14.749 7.08258 14.0644C4.78771 14.5602 4.30278 13.096 4.30278 13.096C3.9285 12.1452 3.38757 11.8953 3.38757 11.8953C2.639 11.3873 3.44358 11.3969 3.44358 11.3969C4.27273 11.4553 4.70849 12.243 4.70849 12.243C5.44339 13.4967 6.63863 13.134 7.10853 12.9221C7.18229 12.3911 7.39676 12.0298 7.63308 11.8247C5.79991 11.6182 3.87386 10.9147 3.87386 7.77176C3.8625 6.95666 4.1667 6.1684 4.72351 5.57008C4.63882 5.36363 4.35605 4.52968 4.8041 3.39692C4.8041 3.39692 5.49667 3.17689 7.07302 4.23766C8.42512 3.87001 9.85169 3.87001 11.2038 4.23766C12.7788 3.17689 13.47 3.39692 13.47 3.39692C13.9194 4.52696 13.6366 5.36091 13.5519 5.57008C14.1105 6.1685 14.4153 6.95813 14.4029 7.77447C14.4029 10.9242 12.4728 11.6182 10.6369 11.8206C10.9319 12.076 11.1956 12.5744 11.1956 13.3405C11.1956 14.4379 11.186 15.3208 11.186 15.5911C11.186 15.8097 11.3349 16.0651 11.7529 15.9849C13.6106 15.3681 15.1871 14.1142 16.2003 12.4476C17.2136 10.781 17.5975 8.81039 17.2834 6.88815C16.9693 4.9659 15.9777 3.21745 14.486 1.95549C12.9943 0.693525 11.0997 0.000372037 9.14113 0H9.1384Z"
                fill="#FCCDCC"
                fill-rule="evenodd"
              />
            </svg>
          </a>
          <a
            class="social linkedin"
            href="https://www.linkedin.com/company/forbole"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              fill="none"
              height="14"
              viewBox="0 0 15 14"
              width="15"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M13.1076 0H1.17459C0.900492 0 0.637624 0.108884 0.44381 0.302697C0.249997 0.496511 0.141113 0.759378 0.141113 1.03347V12.9665C0.141113 13.2406 0.249997 13.5035 0.44381 13.6973C0.637624 13.8911 0.900492 14 1.17459 14H13.1076C13.3817 14 13.6446 13.8911 13.8384 13.6973C14.0322 13.5035 14.1411 13.2406 14.1411 12.9665V1.03347C14.1411 0.759378 14.0322 0.496511 13.8384 0.302697C13.6446 0.108884 13.3817 0 13.1076 0ZM4.31389 11.9262H2.20903V5.24028H4.31389V11.9262ZM3.26 4.31375C3.02124 4.31241 2.78823 4.24036 2.59037 4.10672C2.39252 3.97307 2.23869 3.78381 2.14829 3.56282C2.0579 3.34183 2.03499 3.09901 2.08246 2.86502C2.12994 2.63102 2.24566 2.41633 2.41504 2.24804C2.58441 2.07975 2.79984 1.96541 3.03414 1.91944C3.26843 1.87346 3.5111 1.89793 3.7315 1.98974C3.95191 2.08155 4.14018 2.2366 4.27255 2.43531C4.40492 2.63402 4.47547 2.86749 4.47528 3.10625C4.47753 3.2661 4.44758 3.42477 4.3872 3.5728C4.32683 3.72083 4.23727 3.85519 4.12386 3.96787C4.01046 4.08055 3.87552 4.16925 3.72711 4.22867C3.5787 4.28809 3.41984 4.31703 3.26 4.31375ZM12.0722 11.9321H9.96834V8.27944C9.96834 7.20222 9.51042 6.86972 8.91931 6.86972C8.29514 6.86972 7.68264 7.34028 7.68264 8.30667V11.9321H5.57778V5.24514H7.60195V6.17167H7.62917C7.83236 5.76042 8.54403 5.0575 9.63 5.0575C10.8044 5.0575 12.0732 5.75458 12.0732 7.79625L12.0722 11.9321Z"
                fill="#FCCDCC"
              />
            </svg>
          </a>
        </div>
        <div
          class="terms__wrapper"
        >
          <span
            class="MuiTypography-root MuiTypography-caption emotion-39"
          >
            copyright
          </span>
          <span
            class="MuiTypography-root MuiTypography-caption terms--dash desktop emotion-39"
          >
            |
          </span>
          <div
            class="terms__links"
          >
            <a
              class="MuiTypography-root MuiTypography-caption link__item link__item--caption emotion-39"
              href="/terms-and-conditions"
              target="_self"
            >
              termsAndConditions
            </a>
            <span
              class="MuiTypography-root MuiTypography-caption terms--dash emotion-39"
            >
              |
            </span>
            <a
              class="MuiTypography-root MuiTypography-caption link__item link__item--caption emotion-39"
              href="/privacy-policy"
              target="_self"
            >
              privacyPolicy
            </a>
          </div>
          <span
            class="MuiTypography-root MuiTypography-caption terms--dash desktop emotion-39"
          >
            |
          </span>
          <span
            class="MuiTypography-root MuiTypography-caption emotion-39"
          >
            common:productOf
          </span>
        </div>
      </div>
    </footer>
  </main>
</DocumentFragment>
`;
